### ISSUE TITLE
new Useragent

### ISSUE THREAD

**Issue by wujixian** [nielsbasjes/yauaa#367]
**The exact useragent**
useragent: 'Mozilla/5.0 (Linux; Android 11; V1824A; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/87.0.4280.141 Mobile Safari/537.36 VivoBrowser/10.4.0.2'

**Expected/Incorrect fields**
DeviceBrand: 'V1824A'

...

**Additional information**
hi, bro  i have a new user agent.  the phone brand is IQOO whitch is chniese  smart phone brand, here is the user agent detail :  
 
Mozilla/5.0 (Linux; Android 11; V1824A; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/87.0.4280.141 Mobile Safari/537.36 VivoBrowser/10.4.0.2

now i want add a new matcher, could i add one? like top user agent ,how could i add a correct matcher. and whitch file  i could write in. (eg. Xiaomi.yaml, Mozilla.yaml...) or  create a new yaml file?  
 
![image](https://user-images.githubusercontent.com/11608170/141956643-e27047bc-251c-41ca-99ee-8729ede50597.png)
i could not understand the sentences mean.  could you give me some tips?

**Comment by nielsbasjes** [comment]
Hello,
Thank you for the bug report.
You are correct: the reported brand is incorrect.

I did a search and found that this is a "VIVO IQOO V1824A"

There are too many devices from too many brands to keep a complete list of all of them.
See also https://yauaa.basjes.nl/using/limitations/#device-name-and-device-brand

I did find that only Vivo devices use the VivoBrowser; I'm going to try to make this work.
